lib/group/group-router.js

```diff
@@ -1,56 +1,66 @@
 'use strict';
 
 var express = require('express'),
-  config = require('./config-group');
+  config = require('./config-group'),
+  shortid = require('shortid');
 
-function initRouter(mediator) {
+var GroupCloudDataTopics = require('fh-wfm-mediator/lib/topics');
+
+/**
+ * Function which handles the response according to the promise returned by the topic
+ *
+ * @param {Object} res
+ * @param {Promise} topicPromise Promise returned by the topic the request was sent to
+ */
+function handleResponse(res, topicPromise) {
+  topicPromise.then(function(data) {
+    res.status(200).json(data);
+  }).catch(function(err) {
+    console.log("[Error] Group topic request: ", err);
+    res.status(500).json({error: err.message});
+  });
+}
```

Comment:
This one should be moved somewhere where it's more reusable, else we'll have a copy of it on a lot of routers.

Also `.catch(function(err) {throw err;})` instead of finishing the response so it can get handled by the global express error handler.

This is also a nice opportunity to think with higher-order functions. How would this have to look like if the usage was:
```javascript
router.route('/').get(handlePromiseResponse(function(req, res) {
  return Promise.resolve({some: 'data'}); // and with `return {some: 'data'};`
}));
```